### PR TITLE
Bug/718/queue requests

### PR DESCRIPTION
**Pull request by r-hannuschka**

### CODE REVIEW

File: libs/shared/README.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# shared
```
Comment (konne) [inline libs/shared/README.md:1]: remove this file or make it proper, so write what should be in the shared lib

File: apps/web/tsconfig.app.json [hunk 2]
```diff
@@ -5,7 +5,7 @@
     "outDir": "../../dist/out-tsc",
     "types": [],
     "target": "ES2022",
-    "useDefineForClassFields": false
+    "useDefineForClassFields": false,
```
Comment (konne) [inline apps/web/tsconfig.app.json:8]: why you want to have that change already, just to be future proof?

**Review by konne — APPROVED** [review]
@r-hannuschka please see comments

File: apps/web/tsconfig.app.json (hunk 2 above)
Comment (r-hannuschka) [reply apps/web/tsconfig.app.json:8]: i did not add this i think it comes with the latest migration but we take it to be future proof

File: libs/shared/README.md (hunk 1 above)
Comment (r-hannuschka) [reply libs/shared/README.md]: removed

**Pull request merged**